javascript/node/code-style-recommendations/styling-brackets.md

```diff
@@ -19,14 +15,20 @@ tags:
 aspects:
   - introduction
 
+links:
+  - '[MDN - ASI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Automatic_semicolon_insertion
+){documentation}'
+  - '[MDN - Block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block){documentation}
```

Comment:
The 2nd link is missing a `'` at the end.

Reply:
```suggestion
  - '[MDN - Block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block){documentation}'
```